plugins/mcreator-core/datalists/tabs.yaml

```diff
@@ -1,22 +1,32 @@
+- No creative tab entry:
+  texture: BARRIER
 - BUILDING_BLOCKS:
   texture: BRICK_BLOCK
+  readable_name: Building Blocks
 - DECORATIONS:
-  texture: RED_FLOWER#0
+  texture: GRASS
+  readable_name: Natural Blocks
 - REDSTONE:
   texture: REDSTONE
-- TRANSPORTATION:
-  texture: GOLDEN_RAIL
-- MISC:
-  texture: LAVA_BUCKET
+  readable_name: Redstone Blocks
 - FOOD:
-  texture: APPLE
+  texture: GOLDEN_APPLE
+  readable_name: Food & Drinks
 - TOOLS:
-  texture: IRON_AXE
+  texture: DIAMOND_PICKAXE
+  readable_name: Tools & Utilities
 - COMBAT:
-  texture: GOLDEN_SWORD
-- BREWING:
-  texture: POTIONITEM
+  texture: NETHERITE_SWORD
+  readable_name: Combat
 - MATERIALS:
-  texture: STICK
-- No creative tab entry:
-  texture: BARRIER
\ No newline at end of file
+  texture: IRON_INGOT
+  readable_name: Ingredients
+- MISC:
+  texture: SPAWN_EGG#90
+  readable_name: Spawn Eggs
+- TRANSPORTATION: # not ideal name, but kept for backward compatibility with old workspace and instead added readable_name
+  texture: SIGN
+  readable_name: Functional Blocks
+- BREWING: # not ideal name, but kept for backward compatibility with old workspace and instead added readable_name
+  texture: WOOL#9
+  readable_name: Colored Blocks
```

Comment:
If pre 1.19.3 generators are kept, you should also maybe keep the `Brewing` name and it might confuse some users to see this name for 1.18.2 (or 1.19.2).

Reply:
Generators always target the latest version only, so I was not entirely sure what to do here honseltly

Reply:
Aybe something you can do is: Brewing/Colored Blocks 🤷‍♂️